client/src/pages/Home.js

```diff
@@ -1,15 +1,38 @@
-import React from "react";
+import React, {useEffect} from "react";
 import {useNavigate} from 'react-router-dom';
 import CustomSearchBar from '../components/CustomSearchBar';
 import '../assets/css/home.css';
 import ReviewCardContainer from "../components/ReviewCardContainer";
 import Box from '@mui/material/Box';
 import AddIcon from '@mui/icons-material/Add';
 import IconButton from "@material-ui/core/IconButton";
+import axios from 'axios';
+import ReviewCard from '../components/ReviewCard';
+import {useGlobalState} from '../context/GlobalState';
 
 const Home = (props) => {
+    const [state, setState] = useGlobalState({reviewCards: []});
     const navigate = useNavigate();
 
+    useEffect(async () => {
+        await axios.get(`${window.location.origin.toString()}/review/getAllReviews`, {})
+        .then(res => {
+            let allReviews = [];
+            res.data.data.forEach(x => allReviews.push(
+                <ReviewCard
+                    key={x['_id']}
+                    heading={`${x['User']['UserLogin']}, ${(new Date(x['ReviwedAt'])).toLocaleString('default', { month: 'short', day: 'numeric', year: 'numeric'})} at ${(new Date(x['ReviwedAt'])).toLocaleString('default', { timeStyle: 'long'})}`} 
+                    brandName={x['Product']['ProductBrand']} 
+                    productName={x['Product']['ProductName']} 
+                    review={x['ReviewText']}
+                />));
+            setState({...state, allReviewCards: allReviews, storedReviewCards: allReviews});
```

Comment:
Same question as above what is the purpose of having it stored twice ?

Reply:
can u get on a call thx